trainer/trainer.py

```diff
@@ -1483,10 +1485,33 @@ def _fit(self) -> None:
                 self.save_best_model()
             self.callbacks.on_epoch_end(self)
 
-    def fit(self) -> None:
+    def _fit_with_largest_batch_size(self, starting_batch_size=2048):
```

Comment:
How about making this an explicit call with a separate function like Trainer.fit_with_batch_finder()

Reply:
I like that idea actually. smart